data-jdbc/src/test/groovy/io/micronaut/data/jdbc/oraclexe/jsonview/StudentScheduleClassView.java

```diff
@@ -0,0 +1,66 @@
+package io.micronaut.data.jdbc.oraclexe.jsonview;
+
+import io.micronaut.data.annotation.Embeddable;
+import io.micronaut.data.annotation.Id;
+import io.micronaut.data.annotation.JsonView;
+import io.micronaut.data.annotation.Relation;
+import io.micronaut.data.jdbc.annotation.JoinColumn;
+
+import java.time.LocalTime;
+
+@JsonView(table = "TBL_CLASS")
+@Embeddable
```

Comment:
Do we need both annotations? Can we metadata annotate `@JsonView` with `@Embeddedable`?

Reply:
JsonView is not needed here, this is from some phase of code changes before. Removed and left only `@Embeddable`